### ISSUE TITLE
Is `ReturnValues` supported for BatchWrites?

### ISSUE THREAD

**Issue by cmaycumber** [typedorm/typedorm#200]
Is setting `ReturnValues` currently supported for the batch writer? Or for any other form of write?

**Comment by whimzyLive** [comment]
Hi @cmaycumber ,

It is not supported as of now, but the contributions are always welcomed. 
Let me know if you are interested in implementing it, happy to help with any initial onboarding/setting up. 


Please check the [TypDorm Public API](https://github.com/typedorm/typedorm/blob/main/docs/api.md#entitymanagercreate) for more information.

**Comment by cmaycumber** [comment]
Yeah, would love to help if I can. I'll start to check out some of the codebase.
